apiserver/facades/client/annotations/state.go

```diff
@@ -23,6 +23,6 @@ type stateShim struct {
 	*state.Model
 }
 
-func (s *stateShim) ModelTag() names.ModelTag {
+func (s stateShim) ModelTag() names.ModelTag {
```

Comment:
As stated elsewhere, this is the wrong pattern. We are subverting the semantics of the underlying structs.